dask/dataframe/io/sql.py

```diff
@@ -241,8 +252,11 @@ def to_sql(
     ----------
     name : str
         Name of SQL table.
-    uri : string
-        Full sqlalchemy URI for the database connection
+    con : sqlalchemy.engine.Engine or str
+        Using SQLAlchemy makes it possible to use any DB supported by that
```

Comment:
This comment about sqlalchemy belongs in the text above, not here. Having it here appears to imply that using a str imposes limitations on what you can access, but it does not.

Reply:
ok, renamed this param and updated the docs about it to more clearly describe what it means to us (as opposed to its Pandas counterpart, which differs in some subtle ways)